applications/opentgov.md

```diff
@@ -0,0 +1,200 @@
+| 0c. | Testing | Core functions will be fully covered by comprehensive unit tests to ensure functionality and robustness. |
+| 0d. | Repo | We will provide a repository that can be used to test all the functionality delivered with this milestone. |
+| 0e. | Article | We will publish an **article** that explains what is opentgov and how to start using it |
+| 0f. | Social Media Presence | The @opentgov X account will be used to gather a community of voters. The telegram group will gather a community of voters. |
+| 1. | Production stage | Milestone 1 will include the functional MVP to vote on Telegram using your own Governance Proxy in a self-custodial way. It will come with a detailed documentations explaining how to create a governance proxy & setup the bot. |
+| 1a. | Front-end | Milestone 1 will include a minimalist and clean front-end hosted at opentgov.com to display relevant informations about latest and hottest proposals. |
+| 1b. | Telegram Group | Milestone 1 will include the creation of an automated telegram group with 1 channel per proposal. This place thanks to Telegram UX capabilities will allow efficient discussion between participants. |
+| 1c. | Back-end | Milestone 1 will include the a full self-hosting node.js backend to start voting on proposals directly through telegram with the conviction integrated. |
+| 1d. | Documentation | Milestone 1 will include an exhaustive documentation about how to setup a telegram bot, creating a governance proxy and launching a node program easily. |
+| 2. | Multi-chain & CLI | The Milestone 2 will include the integration of several chains of the DOT ecosystem in addition to Polkadot. It will also come with a CLI tool that will make the setup of the bot even easier than on M1. |
+| 2a. | Substrate Extension | The Milestone 2 will include the capability for the back-end to handle at least 2 other chains part of the Polkadot Ecosystem. |
+| 2b. | Command Line Installer | The Milestone 2 will include the creation of a simple installer. This update will enhance largely the accessibility of the tool for a broader adoption. |
+| 2c. | Feedback collection | The Milestone 2 will include the creation of an user experience survey post-deployment. The goal is to collect feedbacks and improve UX in an upcoming update. |
+
+### Budget Breakdown
+
+| Category | Item | Cost | Amount | Total | Description |
+| --- | ---- | --- | --- | --- | ---|
+| Personel | Full-Stack Developer | 13,000 USD | 1 FTE | 13,000 USD | Developing (frontend and backend) and deploying the project according to the milestones. |
+| Personel | Technical Writing | 2,000 USD | 1 FTE | 2,000 USD | Writing an entire documentation covering telegram bot setup, governance proxy creation and overall project usage. |
+| Equipment | 1 year 32GB RAM VPS | 770 USD | 2 FTE | 1440 USD | VPS to host the backend (telegram group management) and the frontend of the websites (onepager and documentation). |
```

Comment:
Hey @mar1, can you remove the hosting costs from the budget? We specifically excluded these from the program: https://github.com/PolkadotOpenSourceGrants/apply#guidelines